mirrord/operator/src/crd.rs

```diff
@@ -21,7 +21,8 @@ pub const TARGETLESS_TARGET_NAME: &str = "targetless";
 pub struct TargetSpec {
     /// None when targetless.
     pub target: Option<Target>,
-    #[deprecated = "port locks are enforced only in the operator, new operator versions always return an empty list"]
```

Comment:
Why remove this in favor of docs?

Reply:
Too many warnings in the operator code, I got tired of copy-pasting `#[allow(deprecated)]`. But I figured I can just remove the field, it's backwards-compatible ^^

Reply:
Nice!